template/package.json

```diff
@@ -5,7 +5,7 @@
   "author": "{{ author }}",
   "private": true,
   "scripts": {
-    "dev": "webpack-dev-server --inline --progress --config build/webpack.dev.conf.js",
+    "dev": "node build/dev-server.js",
```

Comment:
Hm, that merge doesn't look like it's supposed to ... the webpack-dev-server is new in the `develop`branch, while yours was still built on the old `dev-server` - so the former should stay in and the latter should go out.

But we might simply resolve this during / after the merge ..